README.md

```diff
@@ -33,23 +33,61 @@ Experimental REST API for APEL
     ```
     
 3. Install requirements.txt
+    ```
+    pip install -r requirements.txt
```

Comment:
`MySQL-python` is in `requirements.txt` and the `yum install` arguments. This correct?